rocks-db/src/migrations/offchain_data.rs

```diff
@@ -0,0 +1,23 @@
+use crate::columns::offchain_data::{OffChainData, OffChainDataDeprecated, StorageMutability};
+use crate::migrator::{RocksMigration, SerializationType};
+
+impl From<OffChainDataDeprecated> for OffChainData {
+    fn from(value: OffChainDataDeprecated) -> Self {
+        let immutability = StorageMutability::from(value.url.as_str());
+        Self {
+            storage_mutability: immutability,
+            url: Some(value.url),
+            metadata: Some(value.metadata),
+            last_read_at: 0,
+        }
+    }
+}
+
+pub(crate) struct OffChainDataMigration;
+impl RocksMigration for OffChainDataMigration {
+    const VERSION: u64 = 4;
+    const DESERIALIZATION_TYPE: SerializationType = SerializationType::Flatbuffers;
```

Comment:
looks like this one should be bincode.
Just checked through code. It's one of cbor (insert_gaped_data - luckily never used) or bincode (transaction client and json worker)